actionview/actionview.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activesupport", version
 
   s.add_dependency "builder",       "~> 3.1"
+  s.add_dependency "erubi",         "~> 1.4"
   s.add_dependency "erubis",        "~> 2.7.0"
```

Comment:
Agree that's a poor experience.

Workaround: Have the proxy reference the new constant which is autoloaded from a separate file with an explicit require. Then folks using `ActionView::Template::Handlers::Erubis` will get a straightforward LoadError.

I'll give this a shot and merge.